install_sct.bat

```diff
@@ -28,7 +28,11 @@ echo:
 echo ### Installing SCT and its dependencies...
 rem Pip needs to be upgraded because default p3.7 pip won't resolve dependency conflicts correctly
 python -m pip install --upgrade pip || goto error
-pip install -r requirements.txt || goto error
+if exist requirements-freeze.txt (
+  pip install -r requirements-freeze.txt || goto error
+) else (
+  pip install -r requirements.txt || goto error)
+)

```

Comment:
Maybe should we edit the `echo` to clarify the special case for support and debug purpose ?

```bash
echo ### Installing SCT and its dependencies... (using release frozed requirements)
```

Reply:
Maybe on two lines, allowing you to keep tje generic one outside the condition and adding the detail inside the condition of freeze.

Reply:
Ahh, good idea! :)

Reply:
I've done a barebones version of this in [737ee2e](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3749/commits/737ee2efcccd94b626c5d0e354ba1925a4beb2ee), but maybe we want to be more explicit by using plain English language (like you've done):

> ```
> echo ### Installing SCT and its dependencies... (using release frozed requirements)
> ```

Reply:
I think it's ok. People will understand the basics and we have the info for debugging.